runhouse/resources/hardware/launcher_utils.py

```diff
@@ -180,7 +180,7 @@ def _update_from_den_response(cls, cluster, config: dict):
             "ssh_properties",
             "client_port",
         ]:
-            value = config.get(attribute)
+            value = config.get("cluster_config").get(attribute)
```

Comment:
we should prob leave this as is, and instead parse the response before calling `_update_from_den_response`. is the issue that the config being passed in has two keys (`cluster_config` and `launched_config`)? if so we can just set config=data["cluster_config"] in the `up` method